### PR TITLE
Jess/local

### PR DESCRIPTION
**Pull request by jlotto8**

### CODE REVIEW

File: local-setup.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# nano should be installed
+if (which nano)
+then
+  echo 'nano already installed'
+else
+  echo 'installing nano'
+   brew install nano
```
Comment (stephenlauck) [inline local-setup.sh:7]: There is an extra space here but it's ok, we'll fix it later.

**Review by stephenlauck — APPROVED** [review]

**Review by HarmonyMorse — APPROVED** [review]
Awesomesauce

**Review by whmcquaid — APPROVED** [review]
:shipit:

**Review by DevinBeu — APPROVED** [review]
look fire

**Pull request merged**